packages/trip-form/src/TripOptions/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { FunctionComponent, ReactElement, useState } from "react";
+
+import GeneralSettingsPanel from "../GeneralSettingsPanel";
+
+import FeaturedOption from "./FeaturedOption";
+import FeaturedOptionOverlay from "./FeaturedOptionOverlay";
+import ModeRow from "./ModeRow";
+import TransitOptions from "./TransitOptions";
+import { Company, QueryProps } from "./types";
+import * as S from "./styled";
+
+interface ComponentProps {
+  /**
+   * React element to be rendered below
+   * the rest of the element
+   */
+  footer?: ReactElement;
+
+  /**
+   * List of company objects to include in the
+   * featured options
+   */
+  supportedCompanies: Company[];
+
+  /**
+   * Classnames to add to the container div to allow additional styling
+   */
+  className: string;
+
+  /**
+   * Icon props used for overwriting icons and images throughout the component
+   */
```

Comment:
The JSDoc comment should be added to each field so it appears for each field in Storybook under the "Docs" section of a story.

Reply:
Good call! Added in 775500dafb258ffc2fc5d431b8619ebe3f21b54f